rest-api.md

```diff
@@ -0,0 +1,767 @@
+## API
+
+### /keys
```

Comment:
```suggestion
### Get keys
```

Reply:
This is endpoint name.
I prefer
Create key - post creation key
Get key - get one key
List keys - get list of keys
And similar to this

Reply:
Let's use one describe format everywhere

Smth like this
```
### Endpoint name

What endpoint returns

> :warning: Some warns and other details 

Query:
- param - param details. param type?

Request example
`curl http://123.com/`

Response
```json
success response schema
```